packages/ui/src/lib/spinners/Spinner.stories.svelte

```diff
@@ -0,0 +1,25 @@
+<script>
+	import { Meta, Story } from '@storybook/addon-svelte-csf';
+
+	import Spinner from './Spinner.svelte';
+</script>
+
+<Meta title="Ui/Spinners" component={Spinner} />
+
+<Story name="Default">
+	<div class="text-core-grey-800">
+		<Spinner class="w-40" />
+	</div>
+</Story>
+
+<Story name="Thickness">
+	<div class="text-core-grey-800">
+		<Spinner class="w-40 stroke-[16]" />
+	</div>
+</Story>
+
+<Story name="Color">
+	<div class="text-core-grey-800">
+		<Spinner class="w-40" circleClass="stroke-core-blue-200" arcClass="stroke-core-glaPink-600" />
+	</div>
+</Story>
```

Comment:
An example  showing the spinner working inside a button might be useful:

```suggestion
</Story>

<Story name="In a button">
	<div class="text-core-grey-800">
		<Button>I'm loading...
			<Spinner class="h-6 w-6"/>
		</Button>
	</div>
</Story>
```